lib/scrabble.rb

```diff
@@ -1,4 +1,13 @@
 module Scrabble
+  require "awesome_print"
+  require "pathname"
+  require "csv"
   # require your gems and classes here
   # require_relative 'scrabble/whatevs'
+
+  PROJECT_ROOT = Pathname(__dir__).parent
+
+  require_relative PROJECT_ROOT.join('lib/scrabble/scrabble')
```

Comment:
I think ```require_relative``` should be relative to this file, so all of the PROJECT_ROOT.join should not be needed.